windows/utils/get_windows_system_info.yml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+- include_tasks: ../../common/get_guest_system_info.yml
+
+- include_tasks: win_get_bitness.yml
+- include_tasks: win_get_guest_type.yml
+
```

Comment:
It looks like we can get guest type from ansible_os_product_type:
https://github.com/ansible-collections/ansible.windows/blob/main/plugins/modules/setup.ps1#L610
Except that in ansible.windows collection it is using 'workstation' instead of 'client'.

And for Windows guest, ansible_architecture indicates "32-bit" or "64-bit", so I guess we can also get windows bitness from ansible_architecture. But it is not suitable for Linux, which shows like "x86_64".

Therefore, I suggest we can set win_guest_bitness and win_guest_type directly here as these information has been retrieved in `common/get_system_info.yml`.

Reply:
Yeah, right, it's more convenient here to use ansible facts directly. Thanks.